### PR TITLE
Refactor diagrams and update user guide

### LINKED ISSUE #102: Refactor diagrams
#### Agent Version
1.0+

#### OpenStack Release
Kilo+

#### Description
The diagrams in the user guide need to be refactored for clarity and adherence to F5 branding.

### PR DESCRIPTION
**Pull request by jputrino**
@mattgreene @richbrowne 
#### What issues does this address?
Fixes #102 #80 

#### What's this change do?
-updates diagrams in the user guide for clarity and adherence to F5 branding
-some text edits for clarity and accuracy
-clarified 'agent redundancy'

#### Where should the reviewer start?
Build the docs and view the images inline (we can do this together or I can provide instructions).

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -300,3 +306,11 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/': None}
+
+intersphinx_mapping = {'heat': (
+     'http://f5-openstack-heat.readthedocs.org/en/latest', None),
+     'docs': (
+     'http://f5-openstack-docs.readthedocs.org/en/latest', None),
+     'lbaasv2': (
```
Comment (mattgreene) [inline docs/conf.py]: This PR is for the lbaasv1 repo, so shouldn't this mapping table point to http://f5-openstack-lbaasv1?

**Comment by mattgreene** [comment]
docs/media/f5-lbaas-horizontal-scaleout.png

Need to add a line connecting plugin driver to the external agent via the message queue.

File: docs/conf.py (hunk 1 above)
Comment (jputrino) [inline docs/conf.py]: Nope. This is what allows me to link to docs in other projects.

**Comment by mattgreene** [comment]
docs/_build/map_multiple-agents-same-host.html

step 6.  Need a space between in "sudoervice"

**Comment by mattgreene** [comment]
docs/_build/map_bigip-ha.html

Clustering is coming soon.

**Comment by mattgreene** [comment]
Need to tweak the headings.  Most of the "User Guide" is nested under the "Compatibility" section.  Need to move it up a level.  How about:

Preface (Heading 1)
    Release Version (Heading 2)
    Compatibility (Heading 2)
User Guide (Heading 1)
    Overview (Heading 2)
    ...

screen shot 2016-05-11 at 5 08 42 pm

**Comment by mattgreene** [comment]
Address these minor comments and I think we're good for the set of change.

**Comment by jputrino** [comment]
@mattgreene - should be good to go!

**Pull request merged**